k9mail-library/src/test/java/com/fsck/k9/mail/store/imap/ImapFolderTest.java

```diff
@@ -462,7 +462,7 @@ public void getUnreadMessageCount_withClosedFolder_shouldThrow() throws Exceptio
     public void getUnreadMessageCount_connectionThrowsIOException_shouldThrowMessagingException() throws Exception {
         ImapFolder folder = createFolder("Folder");
         prepareImapFolderForOpen(OPEN_MODE_RW);
-        when(imapConnection.executeSimpleCommand("SEARCH 1:* UNSEEN NOT DELETED")).thenThrow(new IOException());
```

Comment:
I'd rather see us add tests than change ones if it's new behaviour. Do we never use non-UID based search now?